ocdata/core/bulk.py

```diff
@@ -0,0 +1,287 @@
+import os
+import pickle
+
+import ase
+import numpy as np
+import pymatgen
+from pymatgen.core.surface import (
+    SlabGenerator,
+    get_symmetrically_distinct_miller_indices,
+)
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
+
+from ocdata.configs.constants import COVALENT_MATERIALS_MPIDS, MAX_MILLER
```

Comment:
Let's remove MAX_MILLER from the constants. This is a hard-coded artifact of OC20.

Reply:
Incorporated here: https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset/blob/refactor-v2/ocdata/core/bulk.py#L62.